internal/scheduler/api.go

```diff
@@ -52,10 +55,10 @@ func (srv *ExecutorApi) StreamingLeaseJobs(stream api.AggregatedQueue_StreamingL
 	}
 
 	// Get leases assigned to this executor.
-	queries := New(srv.Db)
+	queries := sqlc.New(srv.Db)
```

Comment:
the `sqlc` directory here is probably temporary.  I've moved the database stuff here partly to avoid cyclic references and partly because calling `scheduler.New()` didn't create a new scheduler, but rather just the database and I found that confusing.

One of the next taks is to start using database migrations for this database so I expect this will change again at that point.

Reply:
Using sqlc as a package name is a bit confusing I think. Since we import another package named sqlc.

Reply:
Yeah I'll fix this in the next pr